rosdep/base.yaml

```diff
@@ -1673,6 +1673,20 @@ libboost-program-options-dev:
   openembedded: [boost@openembedded-core]
   ubuntu: [libboost-program-options-dev]
 libboost-python:
+  alpine: [boost-python2]
+  debian:
+    buster: [libboost-python1.67.0]
+    jessie: [libboost-python1.55.0]
+    stretch: [libboost-python1.62.0]
+  fedora: [boost-python2-devel, boost-python3-devel]
+  ubuntu:
+    bionic: [libboost-python1.65.1]
+    disco: [libboost-python1.67.0]
+    eoan: [libboost-python1.67.0]
+    focal: [libboost-python1.67.0]
```

Comment:
This looks outdated: https://packages.ubuntu.com/focal/libboost-python1.71.0

Reply:
It was still in "proposed' when I opened the PR. Updated in https://github.com/ros/rosdistro/pull/23844/commits/d6effc59f2ba003d5aef6fb4f14598dd007c2964